.changes/next-release/bugfix-25a96fb5-3f37-4583-a502-63c6dd1a42bc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix to pass context specific user intent and trigger type to Amazon Q chat"
```

Comment:
don't think we'd need a changelog update for minor internal bug fix.

Reply:
Changelog update removed.